ProcessLib/GlobalVectorFromNamedFunction.cpp

```diff
@@ -53,10 +54,11 @@ GlobalVector const& GlobalVectorFromNamedFunction::call(
         _context.index = node_id;
         auto const value = _function_caller.call(args);
 
-        // TODO Problems with PETSc? (local vs. global index)
+        // Problems with PETSc also uses global index.
```

Comment:
Dropped. This just answers the deleted comment (original line 56), and I should write it as a comment in this PR.